assets/css/pages/archive.less

```diff
@@ -8,6 +8,21 @@ main.archive {
     }
   }
 
+  h1 .featured {
+    // bullshit CSS because tara needs to take a pass at this
```

Comment:
I plan to spend some time in the near future doing a bullshit CSS pass, so worry not